XMonad/Hooks/WallpaperSetter.hs

```diff
@@ -100,9 +100,16 @@ defWallpaperConf = WallpaperConf "" $ WallpaperList []
 instance Default WallpaperConf where
     def = defWallpaperConf
 
--- |returns the default association list (maps name to name.jpg, non-alphanumeric characters are omitted)
 defWPNames :: [WorkspaceId] -> WallpaperList
-defWPNames xs = WallpaperList $ map (\x -> (x,WallpaperFix (filter isAlphaNum x++".jpg"))) xs
+defWPNames = defWPNamesJpg
```

Comment:
You should add something like a 

``` haskell
{-# DEPRECATED defWPNames "Use defWPNamesJpg instead" #-}
```

here instead

Reply:
Thanks for pointing out. Should I change the CHANGES.md for that? Like `defWPNames` is deprecated?

Reply:
Yeah we usually include this in the `CHANGES.md`, good thinking :)